src/scripts/main.js

```diff
@@ -207,6 +223,24 @@ topsectionForm.addEventListener('submit', (event) => {
     });
 });
 
+function bubbleSortConcept2(arr) {
+  let swapped;
+
+  do {
+    swapped = false;
+    arr.forEach((item, index) => {
+      if (index < arr.length - 1) {
+        if (item['name'] > arr[index + 1]['name']) {
+          const temp = item;
+          arr[index] = arr[index + 1];
+          arr[index + 1] = temp;
+          swapped = true;
+        }
+      }
+    });
+  } while (swapped);
+}
```

Comment:
The name of the function could be reduced to bubbleSort.
It is good to avoid return of undefined. Also, this function mutates the array. It is difficult to test such functions. Also it may cause unexpected effects in js.